### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Stanford997**
## Context

Update README

## Changes

>- Update README.md

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by adamma1024 — CHANGES_REQUESTED** [review]
Please change your PR title and check if all of files needs to be pushed. Personally, I don't think the files under .idea/ should be included here.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,62 @@
 # CVCoach
 
+This repository is a project for METCS673. This project focuses on using AI to automate and improve the process of resume evaluation and interview preparation. By incorporating Retrieval-Augmented Generation (RAG), we ensure that our application can provide more accurate, context-relevant reviews.
+
 An artifical intelligence tool powered by the ???. It can provide several services for job seekers, interviewers and HRs, such as refining the CV, mocking interviews and writing questions based on resumes.
 
+## Data Collection and preprocessing
```
Comment (adamma1024) [inline README.md:5]: The details of implementation should be moved into a separated file, like the `doc/development_manual.md` did. But you can do it later.

**Review by adamma1024 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Stanford997) [reply README.md:5]: Sure. I will do it later.

**Review by adamma1024 — APPROVED** [review]

**Review by adamma1024 — DISMISSED** [review]

**Review by adamma1024 — APPROVED** [review]

**Comment by adamma1024** [comment]
I closed this PR because it seems like having issues.

**Pull request closed without merging**